demo-dotnet/QuantizationAndStorageOptions/README.md

```diff
@@ -0,0 +1,54 @@
+---
+page_type: sample
+languages:
+  - python
+name: Vector quantization and storage options in C#
+products:
+  - azure
+  - azure-ai-search
+description: |
+  Using Azure.Search.Documents and the Azure SDK for C#, save on storage when using vector quantization
+urlFragment: quantization-storage-c#
+---
+
+# Vector quantization and storage options using C# (Azure AI Search)  
+
+The C# program creates vectorized data on Azure AI Search and demonstrates how to save on storage using the following features
+
+- Quantization
+- Disabling storage of vectors returned in query responses. These vectors are stored separately from vectors used for the queries themselves.
+- Smaller data types than `Edm.Single`.
+
+The sample data is a JSON file containing a pre-chunked version of the sample documents. They have been embedded using text-embedding-3-large with 3072 dimensions.
```

Comment:
line 22 -- can you say anything more about the sample data? Is it hotels, or ???

Reply:
contoso